app/Http/Requests/RclMessageRequest.php

```diff
@@ -88,9 +88,9 @@ public function withValidator($validator)
                 /** Entry fix time requirement */
                 if (config('app.rcl_time_constraints_enabled') && strlen($this->entry_time) == 4) {
                     if (!$this->entryTimeWithinRange($this->entry_time)) {
-//                        if (config('app.rcl_auto_acknowledgement_enabled')) {
-//                            $this->cpdlcService->sendMessage(author: DatalinkAuthorities::SYS, recipient: $this->callsign, recipientAccount: Auth::user(), message: sprintf(RclResponsesEnum::Contact->value, strtoupper(DatalinkAuthorities::OCEN->description())), caption: RclResponsesEnum::Contact->text());
-//                        }
+                        if (config('app.rcl_auto_acknowledgement_enabled')) {
+                            $this->cpdlcService->sendMessage(author: DatalinkAuthorities::SYS, recipient: $this->callsign, recipientAccount: Auth::user(), message: sprintf(RclResponsesEnum::Contact->value, strtoupper(DatalinkAuthorities::OCEN->description())), caption: RclResponsesEnum::Contact->text());
```

Comment:
i'm still confused why this is here. isn't this the validator method? do we want to send this message if things don't validate?

Reply:
That's something I was also thinking about. It seems the IRL implementation is to send that message back, which I'd be ok with just doing, but also VATSIM pilots if not stopped forcefully will send bogus requests that will have to be rejected anyway. The message there for 'realism', the validation there to stop bogus requests.